### PR TITLE
OCPBUGS-33030: [release-4.13][manual]Reduce number of reboots in offline tests

### PR DESCRIPTION
**Pull request by mrniranjan**
OCPBUGS-29595: Reduce number of reboots in offline tests (#940)

* Reduce number of reboots in offline tests

Currently in offline tests we rever to original profile after each tests, this is very costly and causing tests to run for long time.

This patch addresses by reverting to the profile after all offline tests are run.

### CODE REVIEW

File: test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go [hunk 1]
```diff
@@ -1566,17 +1561,9 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
...
-				numaInfo, err := nodes.GetNumaNodes(&node)
-				Expect(err).ToNot(HaveOccurred())
-				if len(numaInfo) < 2 {
-					Skip(fmt.Sprintf("This test need 2 NUMA nodes.The number of NUMA nodes on node %s < 2", node.Name))
-				}
-			}
-			for _, node := range workerRTNodes {
-				numaCoreSiblings, err = nodes.GetCoreSiblings(&node)
+			numaTopology := copyNumaCoreSiblings(numaCoreSiblings)
+			if len(numaTopology) < 2 {
```
Comment (yanirq) [inline test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go:1565]: this is not aligned as the 4.14 code (comparing https://github.com/openshift/cluster-node-tuning-operator/pull/1035/files)

Comment (mrniranjan) [reply test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go]: @yanirq addressed the above in latest commit.

**Pull request merged**